.github/workflows/parametric-tests.yml

```diff
@@ -53,6 +53,6 @@ jobs:
           pip install wheel
       - name: Run
         run: |
-          cd system-tests/parametric
           pip install -r requirements.txt
+          cd system-tests/parametric
```

Comment:
Looks like this change is breaking causing the error here https://github.com/DataDog/dd-trace-go/actions/runs/5080981287/jobs/9128695527?pr=2002 ?